test/Toggle/GetActivatedTogglesTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Vovaxs
+ * Date: 24/08/2015
+ * Time: 10:31
+ */
+
+namespace Clearbooks\Labs\Toggle;
+
+
+use Clearbooks\Labs\Toggle\Entity\ActivatedToggleStub;
+use Clearbooks\Labs\Toggle\Gateway\ActivatedToggleGatewayDummy;
+use Clearbooks\Labs\Toggle\Gateway\ActivatedToggleGatewayStub;
+
+class GetActivatedTogglesTest extends \PHPUnit_Framework_TestCase
+{
+
+    public function setUp()
+    {
+
+    }
```

Comment:
If it's going to be filled later then create it later. If you do this and leave it empty then it doesn't call the parent setUp which could cause potential issues in future